.github/workflows/test-build.yml

```
@@ -0,0 +1,13 @@
+name: Test and build
+on: [push]

```

I have found it useful to run only on pull request, or push to master, so it doesn't run on every push to any branch, but push to any branch also works. Up to you, but sharing this example of how to do this if you want it: 
```suggestion
on:
  push:
    branches:
      - master
  pull_request:
```

---

Thanks, @leighmcculloch ! We used to run it on every push. If it gets annoying, we can change to use only master.